pymc/aesaraf.py

```diff
@@ -49,10 +49,10 @@
 )
 from aesara.graph.fg import FunctionGraph
 from aesara.graph.op import Op
-from aesara.sandbox.rng_mrg import MRG_RandomStream as RandomStream
 from aesara.scalar.basic import Cast
 from aesara.tensor.basic import _as_tensor_variable
 from aesara.tensor.elemwise import Elemwise
+from aesara.tensor.random import RandomStream
```

Comment:
Can we remove the whole `at_rng` and `set_rng` related utilities below from this file?

https://github.com/pymc-devs/pymc/blob/12f4cd73bae95c42a28c2f07f6aab60071fd2ca1/pymc/aesaraf.py#L872-L912

Reply:
We can, but it's only used by VI